doc/design/mount-config-design.md

```diff
@@ -58,7 +58,7 @@ mount: [
 
 | Name | Value |
 | -- | -- |
-| target | This is the target mount point within the TEE. |
+| target | This is the target mount point within the TEE. The target mount point needs to already exist in the target filesystem for the mount to succeed. |
```

Comment:
This is not clear to me. Do you mean the **path** of the mount point needs to already exist?